OpenRA.Test/OpenRA.Game/FluentTest.cs

```diff
@@ -0,0 +1,37 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using NUnit.Framework;
+
+namespace OpenRA.Test
+{
+	[TestFixture]
+	public class FluentTest
+	{
+		readonly string pluralForms = @"
```

Comment:
Can we use the file instead of this?

Reply:
we should use both perhaps?

Reply:
What is the file used for?

Reply:
It is a temporary file as I required syntax highlighting to fix an error of mine.